bitnami/kibana/templates/deployment.yaml

```diff
@@ -0,0 +1,186 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "kibana.fullname" . }}
+  labels: {{- include "kibana.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{- tpl (toYaml .Values.updateStrategy) $ | nindent 4 }}
+  {{- end }}
+  selector:
+    matchLabels: {{- include "kibana.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{- include "kibana.labels" . | nindent 8 }}
+    spec:
+    {{- if .Values.schedulerName }}
+      schedulerName: {{ .Values.schedulerName | quote }}
+    {{- end }}
+{{- include "kibana.imagePullSecrets" . | indent 6 }}
+    {{- if .Values.securityContext.enabled }}
+      securityContext:
+        fsGroup: {{ .Values.securityContext.fsGroup }}
```

Comment:
don't we need the `runAsUser` too?

Reply:
That goes in the container spec, so we allow root containers like the volumePermissions one

Reply:
right